packages/apollo-cli/src/commands/queries/extract.ts

```diff
@@ -0,0 +1,80 @@
+import 'apollo-codegen-core/lib/polyfills';
```

Comment:
What’s this polyfill relatated to? Might be worth putting a comment about its necessity/purpose since it’s imported in its entirety rather generically.

Reply:
I'm not sure why it's included myself. I'll dig around and see. My first guess is lack of native support for the functions in Node 8